tests/tests/object_store/add_put.rs

```diff
@@ -59,6 +59,50 @@ macro_rules! common_tests {
             let res = store.$method(1u8).with_key(expect_str.as_str()).await;
             assert_eq!(res, Ok(expect_str));
         }
+
+        #[wasm_bindgen_test]
```

Comment:
1. This [breaks tests that run without the `serde` feature](https://github.com/Alorel/rust-indexed-db/actions/runs/13884231142?pr=50) - the tests should be runnable with the `wasm-pack test --headless --chrome --firefox` command (or `--firefox`). I _think_ the best approach here is to just add a `#[cfg(feature = "serde")]` flag on this test fn
2. Not sure the name fits the test? You're checking whether deeply nested objects are serialised correctly

Reply:
Aaand that's the tests patched up. Thanks again for the PR!